scenarios/asset-store-migration/8-vote.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Please remove `sudo` in that script. I am part of `docker` group so I don't need sudo and you are calling the script with sudo in the README.